### PR TITLE
update cldr population link

### PR DESCRIPTION
**Pull request by ImUrX**
do I need to run something? I modified this from Github

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -858,7 +858,7 @@ class Direction(models.TextChoices):
         default=0,
         help_text="""
         Number of native speakers. Find locale code in CLDR territoryInfo.json:
-        https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/territoryInfo.json
+        https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-core/supplemental/territoryInfo.json
```
Comment (mathjazz) [inline pontoon/base/models.py:861]: Please remove the trailing space.

**Comment by mathjazz** [comment]
Sorry, I missed that earlier. This is a change to the DB schema, so it requires a DB migration. Could you please create one?

**Comment by ImUrX** [comment]
Yeah, I need to setup the docker env and run the command. I don't have a good internet currently so the docker part will take it's time :pensive:

**Comment by mathjazz** [comment]
> Yeah, I need to setup the docker env and run the command. I don't have a good internet currently so the docker part will take it's time 😔

I see. If you want, I can take it from here.

**Comment by ImUrX** [comment]
Please do, and thank you.

**Review by mathjazz — APPROVED** [review]

**Comment by mathjazz** [comment]
Thanks for the patch, @ImUrX!

**Pull request merged**